### ISSUE TITLE
Which odometry topic is the one that produced the translation and rotation errors shown in the KITTI website?

### ISSUE THREAD

**Issue by claydergc** [laboshinl/loam_velodyne#88]
In the code there are 3 odometry topics:

/laser_odom_to_init
/aft_mapped_to_init
/integrated_to_init

According to what I understand, /integrated_to_init uses /laser_odom_to_init and /aft_mapped_to_init to compute the odometry. So, should I suppose /integrated_to_init is the most accurate and the one used in KITTI?

Thanks.

**Comment by andre-nguyen** [comment]
No, the after mapped is the most accurate as it has the additional step of doing scan to map matching.

**Comment by Li-Shang** [comment]
But aft_mapped_to_init has fewer transformation data compared to laser_odom_to_init. 
So, how to output the transformation of every frame?

Thanks.
